client/v2/common/models/error_response.go

```diff
@@ -3,7 +3,7 @@ package models
 // ErrorResponse an error response with optional data field.
 type ErrorResponse struct {
 	// Data
-	Data *map[string]interface{} `json:"data,omitempty"`
+	Data string `json:"data,omitempty"`
```

Comment:
I'm confused why this changed...

Reply:
this is generated by the code generator. This is the struct in algod, https://github.com/algorand/go-algorand/blob/master/daemon/algod/api/server/v2/generated/types.go#L317